dashboard/test/ui/features/teacher_homepage.feature

```diff
@@ -78,6 +78,33 @@ Feature: Using the teacher homepage sections feature
     Then I should see the section table
     And the section table row at index 0 has primary assignment path "/s/coursea-2018"
 
+  Scenario: Update a section
+    When I see the section set up box
+    And I create a new section with course "Computer Science Principles", version "'18-'19 (Recommended)" and unit "CSP Unit 1 - The Internet"
+    Then the section table should have 1 rows
+    And the section table row at index 0 has primary assignment path "/courses/csp-2018"
+    And the section table row at index 0 has secondary assignment path "/s/csp1-2018"
+
+    When I click selector ".ui-test-section-dropdown"
+    And I click selector ".edit-section-details-link"
+    And I wait until element "#uitest-section-name" is visible
+
+    # Section name input field needs to be emptied before typing new name
+    And I type "" into "#uitest-section-name"
```

Comment:
thanks for this comment 👍 I was wondering why you needed to type the empty string first when you mentioned this combo earlier